mm2src/mm2_main/src/lp_ordermatch/simple_market_maker.rs

```diff
@@ -113,7 +112,7 @@ impl From<std::string::String> for OrderProcessingError {
 #[derive(Deserialize)]
 pub struct StartSimpleMakerBotRequest {
     cfg: SimpleMakerBotRegistry,
-    price_url: Option<String>,
```

Comment:
Could we please leave the singular `price_url` for backwards compatibility in the short term? e.g. translate it into a list with single item?
During netid migration it would be optimal to avoid any maker config issues until transition is complete.

Reply:
No error is thrown if using the old param, looks like it uses the defaults. I'll can mark it as deprecated in docs, could we also add a log message in the bot update loop when it is used to warn users of the deprecation?

Reply:
> could we also add a log message in the bot update loop when it is used to warn users of the deprecation?

Mmm, I guess to do that I have to keep `price_url` in addition to `price_urls` to check if it's used or not. If used, what would you like the behaviour to be of the 2 below options:
1 -  Translate it into a list with single item and log a warning message.
2 - Use default list and log warning message.

Reply:
A third option is to revert `price_urls` to `price_url` and make it accept either a list or a string, but it won't be very good in my opinion since we need to inform users about the change.

Reply:
> A third option is to revert price_urls to price_url and make it accept either a list or a string, but it won't be very good in my opinion since we need to inform users about the change.

I would support both at the same time with an enum.

e.g.,:

```rs
enum PriceSourceType {
    #[serde(rename = "price_url")]
    Singular(String),
    #[serde(rename = "price_urls")]
    Multiple(Vec<String>),
}

impl PriceSourceType {
    /// Always use this to get the data
    fn get_urls(&self) -> Vec<String> {
        match self {
            // TODO: deprecate price_url soon and inform the users
            PriceSourceType::Singular(t) => vec![t.clone()],
            PriceSourceType::Multiple(t) => t.clone(),
        }
    }
}
```

Reply:
Done